.github/fabricbot.json

```diff
@@ -1133,6 +1133,127 @@
       ]
     }
   },
+  {
+    "taskType": "trigger",
+    "capabilityId": "IssueResponder",
+    "subCapability": "PullRequestResponder",
+    "version": "1.0",
+    "config": {
+      "conditions": {
+        "operator": "and",
+        "operands": [
+          {
+            "name": "isOpen",
+            "parameters": {}
+          },
+          {
+            "name": "hasLabel",
+            "parameters": {
+              "label": "no recent activity"
+            }
+          },
+          {
+            "operator": "not",
+            "operands": [
+              {
+                "name": "labelAdded",
+                "parameters": {
+                  "label": "no recent activity"
```

Comment:
Should we just exclude all of the labelAdded events (not just the "no recent activity" one)? I would consider a label applied to a PR shouldn't really be counted as an activity, and instead only remove it when there is a commit pushed or a comment is added.

Reply:
I'm not sure -- adding a label but not closing the issue is a likely indication that we don't want it to auto-close the issue. Note that this rule merely resets the `no recent activity` countdown, it doesn't remove the `needs-author-action` label.

Reply:
Also, this mirrors the logic we're already using in issues. I haven't had any indication so far that filtering out specific events is necessary.